app/dts/bindings/zmk,combos.yaml

```diff
@@ -20,3 +20,5 @@ child-binding:
       default: 50
     slow-release:
       type: boolean
+    layers:
+      type: array
```

Comment:
I think setting a default of -1 would be nice.

Reply:
Hmmm the default is managed by `.layers = DT_PROP_OR(n, layers, {-1}),`. That works for me.